src/sentry/tasks/base.py

```diff
@@ -163,6 +164,7 @@ def wrapped(*args, **kwargs):
             except ignore:
                 return
             except ignore_and_capture:
+                sentry_sdk.set_level("info")
```

Comment:
Is there any benefit to inlining this into the `capture_exception` call as a kwarg vs setting it above the call?

Reply:
kinda? It's cleaner, brain too skateparkge though. [Thread here](https://sentry.slack.com/archives/C0658KY7AFK/p1742842612302899)
Tl:dr decorators are magic and we can pass in the level to the `capture_exception` like so
```
sentry_sdk.capture_exception(level="info") 
```